todo.js

```diff
@@ -0,0 +1,14 @@
+let inputs = document.getElementById("inp")
+
+let text = document.querySelector(".text")
+
+function Add(){
+
+
+    newitem = document.createElement("ul")
+    newitem.innerHTML = `${inputs.value}`
+    text.appendChild(newitem)
+    inputs.value= ""
+
+
+}
```

Comment:
1. Lot's of unncessary empty line. Let's use prettier to handle these things for us.
2. Function name generally follows `camelCase` convention in JS.
3. The `ul` can be added in the HTML file and then targeted in JS file just like the inputs and buttons. Then we'll create a `li` item, place our new todo value, and add it as a child of the `ul`.
4. Let create a independent function for resetting input field. This will help us down the line when we'll need to use it in multiple places.

```suggestion
function addTodo(payload){
    const { value } = payload
    
    newTodo = document.createElement("li")
     
    newTodo.innerHTML = value
    todos.appendChild(newTodo)
    resetInputField()
}
```